### ISSUE TITLE
Fine-tune on Scraped Data

### ISSUE THREAD

**Issue by mattmotoki** [HACC2018/ohia.ai#6]
Fine-tune a keras model with a TensorFlow backend (either trained on ImageNet or our PlantNet/Oxford Flower Datasets) using the labels images and labels from data collection.

* create a generator to stream in training data
    * load data in batches
    * perform data augmentation
    * parallelized (optional)
* replace final layer with a fully connected layer with number of neurons equal to the number of plant categories. 
* freeze all layers except the last
* train the model with early stopping
* save trained model in TensorflowJS format
